2-mandatory/3-financial-times.js

```diff
@@ -5,7 +5,13 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+    let newArticle = [];
```

Comment:
The implementation here is good. A couple of minor points - indentation in the body of the if-statement would improve readability.
Also, maybe the variable name `newArticle` could be better, as it is an array that will hold multiple articles.